### ISSUE TITLE
lint: add semantical unconvert for untyped constants

### ISSUE THREAD

**Issue by Quasilyte** [PieselBois/kfulint#75]
It's hard to solve issue in general, but for well-known cases we can detect unnecessary conversions.

For example:
```go
time.Duration(4) * time.Second
=>
4 * time.Second
```
There is [unconvert](https://github.com/mdempsky/unconvert) already, but it does not detect such cases at this moment.

**Comment by Quasilyte** [comment]
The idea mainly comes from https://github.com/cristaloleg/go-advices#code.
